### PR TITLE
Conditional event

### LINKED ISSUE #62: Conditional props - condition event

### PR DESCRIPTION
**Pull request by barmac**
Closes #62 

image

image

### CODE REVIEW

**Comment by barmac** [comment]
This includes changes from https://github.com/bpmn-io/bpmn-properties-panel/pull/58

File: src/provider/camunda-platform/properties/ConditionProps.js [hunk 1]
```diff
@@ -65,68 +70,26 @@ function ConditionType(props) {
...
-
-      if (source.businessObject.default === businessObject) {
-        commands.push({
-          cmd: 'element.updateProperties',
-          context: {
-            element: source,
-            properties: { 'default': undefined }
-          }
-        });
-      }
```
Comment (barmac) [inline src/provider/camunda-platform/properties/ConditionProps.js]: This is handled in bpmn-js

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

**Review by pinussilvestrus — COMMENTED** [review]
Works good!

In the old properties panel, we have two more properties for conditional catch events

image

which are for some reasons part of the[ old bpmn provider](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/lib/provider/bpmn/parts/implementation/ConditionalEventDefinition.js#L31), although they are camunda related. We will add these in another step (EventProps, cf. #2)

**Comment by barmac** [comment]
That's correct. Other properties for the conditional event will need to be implemented separately.

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**